packages/ui-core/shared/src/lib/components/avatar/avatar.component.scss

```
@@ -2,7 +2,12 @@
 
 @mixin common {
   .inner-wrapper {
-    border-radius: var(--button-rectangle-border-radius);
+    // Deliberately a STADIUM, not the button radius. This is the avatar+name
```

Stylelint (17.14.1)

[error] Unknown word Deliberately (CssSyntaxError)

(CssSyntaxError)